ui/src/components/AlarmFilters.vue

```diff
@@ -8,14 +8,20 @@ import MarkComplete from '@featherds/icon/action/MarkComplete'
 import { FeatherIcon } from '@featherds/icon'
 import { useSituationsStore } from '@/store/useSituationsStore'
 import { sendActionMultiplyAlarms } from '@/services/AlarmService'
-import { removeAlarmsFromSituation } from '@/services/AlecService'
+import {
+	removeAlarmsFromSituation,
+	assignAlarmsToSituation
+} from '@/services/AlecService'
 import { FeatherCheckbox } from '@featherds/checkbox'
 import FiltersSeverity from '@/components/FiltersSeverity.vue'
 import CheckCircle from '@featherds/icon/action/CheckCircle'
-import Delete from '@featherds/icon/action/Delete'
+import ExitToApp from '@featherds/icon/action/ExitToApp'
+import DrawerSituations from '@/components/DrawerSituations.vue'
 import { useAppStore } from '@/store/useAppStore'
+import Delete from '@featherds/icon/action/Delete'
```

Comment:
I found out about `markRaw` recently that maybe worth using. Check it out!
https://vuejs.org/api/reactivity-advanced.html#markraw
https://feather.nanthealth.com/Components/Icon/